app/controllers/health_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class HealthController < ApplicationController
+  skip_before_action :authenticate
+
+  def index
+    render json: { status: 'OK' }
```

Comment:
In `production` I can render the index route even if there are pending migrations.

Reply:
That’s good to know – didn’t know that behaviour is different in prod and development. I added an explicit check for a healthy database connection and pending migrations.